src/features/colinks/wizard/WizardList.tsx

```diff
@@ -37,7 +30,6 @@ export const WizardList = ({ progress }: { progress: WizardProgress }) => {
       }}
     >
       <Step label="Connect Wallet" test={!!address} />
-      <Step label="On Optimism" test={onCorrectChain} />
```

Comment:
How do we know if your ever on the right chain?

Reply:
when you try to mint or buy a link you will be asked to switch networks.
if you don't have enough balance it will till you that you only have xx balance of Optimism Eth and so

Reply:
although once you have balance you will be asked to switch to chain to min at CoSoulButton.tsx line 125
`  if (chain && !onCorrectChain) {
    return (
      <Button color="cta" size="large" onClick={safeSwitchToCorrectChain}>
        Switch to {chain.chainName} to Mint
      </Button>
    );
  }
`